app/assets/src/components/views/sample_page/SamplePage.jsx

```diff
@@ -0,0 +1,328 @@
+import React from "react";
+import cx from "classnames";
+import { get } from "lodash/fp";
+import { getURLParamString } from "~/api";
+import PropTypes from "~/components/utils/propTypes";
+import { pipelineHasAssembly } from "~/components/utils/sample";
+import AMRView from "~/components/AMRView";
+import BasicPopup from "~/components/BasicPopup";
+import PipelineSampleReport from "~/components/PipelineSampleReport";
+import ViewHeader from "~/components/layout/ViewHeader";
+import NarrowContainer from "~/components/layout/NarrowContainer";
+import Tabs from "~/components/ui/controls/Tabs";
+import SampleDetailsSidebar from "~/components/views/report/SampleDetailsSidebar";
+import SampleControls from "./SampleControls";
+import PipelineVersionSelect from "./PipelineVersionSelect";
+
+import cs from "./sample_view.scss";
+
+class SampleView extends React.Component {
```

Comment:
nit: Should it all be called SampleView in `views/SampleView` as a folder? A bit confused about the naming...